src/Symfony/Component/Yaml/Parser.php

```diff
@@ -62,7 +62,7 @@ public function parse($value, $exceptionOnInvalidType = false, $objectSupport =
         $value = $this->cleanup($value);
         $this->lines = explode("\n", $value);
 
-        if (function_exists('mb_internal_encoding') && ((int) ini_get('mbstring.func_overload')) & 2) {
+        if (2 & (int) ini_get('mbstring.func_overload')) {
```

Comment:
It would be easier for auditing purposes to use `MB_OVERLOAD_STRING` instead of `2` in these instances.